packages/website/components/Sidebar.tsx

```diff
@@ -35,6 +39,50 @@ interface Props {
   currentPage?: string;
 }
 
+const components: Array<SidebarSectionType | SidebarLinkType> = [
+  {
+    type: 'section',
+    links: sidebarLinks.layoutComponents,
+    title: 'Layout Components',
+  },
+  {
+    type: 'section',
+    links: sidebarLinks.typographyComponents,
+    title: 'Typography Components',
+  },
+  {
+    type: 'section',
+    links: sidebarLinks.buttonComponents,
+    title: 'Button Components',
+  },
+  {
+    type: 'section',
+    links: sidebarLinks.formComponents,
+    title: 'Form Components',
+  },
+  {
+    type: 'section',
+    links: sidebarLinks.dateComponents,
+    title: 'Date Components',
+  },
+  {
+    type: 'section',
+    links: sidebarLinks.modalComponents,
+    title: 'Modal Components',
+  },
+  {
+    type: 'section',
+    links: sidebarLinks.cardComponents,
+    title: 'Card Components',
+  },
+  {
+    type: 'section',
+    links: sidebarLinks.skeletonComponents,
+    title: 'Skeleton Components',
+  },
+  ...sidebarLinks.unassigned,
```

Comment:
My personal option is that it doesn't make sense, because they are not deprecated but completely removed in v4. All information should be available in `v3 => v4` migration guide and the navigation should focus on v4 only

Reply:
Well, if someone was using those components, then they expect to see them in the navigation with information regarding how to migrate them. That was the feedback that we’ve got during internal interviews with engineers.

Reply:
Was this feedback consistent (based on more than 2 opinions)? I haven't seen this pattern before and usually people just refer to a migration document.

Reply:
Well, consistent with what? Their shared their feedback regarding the components that they expected to see there. Having "deprecated" tag next to the component in the nav is pretty standard I think. Especially that we didn't warn about removal in v3. I really think we should keep those items. but of course we can discuss it with the team when we are back in new year.